DashSync/shared/Models/CoinJoin/DSCoinJoinWrapper.m

```diff
@@ -86,6 +77,30 @@ - (CoinJoinClientOptions *)createOptions {
     return options;
 }
 
+- (void)processDSQueueFrom:(DSPeer *)peer message:(NSData *)message {
+    @synchronized (self) {
+        ByteArray *array = malloc(sizeof(ByteArray));
+        array->len = (uintptr_t)message.length;
+        array->ptr = data_malloc(message);
+        
+        process_ds_queue(_clientManager, peer.address.u8, peer.port, array);
+        
+        if (array) {
+            if (array->ptr) {
+                free((void *)array->ptr);
+            }
+            
+            free(array);
+        }
+    
+        DSLog(@"[OBJ-C] CoinJoin: call");
+        Balance *balance = [self.manager getBalance];
+        
+        run_client_manager(_clientManager, *balance);
+        free(balance);
+    }
```

Comment:
This will work for now for development, but I'll probably have to come up with a better way to run this in parallel for multi-session in future, either by dispatching each session in their own queue or using `Arc<Mutex>> in Rust